### PR TITLE
[zh] Add concepts/signals/_index.md

### PR DESCRIPTION
**Pull request by my-git9**
content/zh/docs/concepts/signals/_index.md

### CODE REVIEW

File: content/zh/docs/concepts/signals/_index.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+title: 信号
+description: 了解 OpenTelemetry 支持的遥测类别
+aliases:
+  - /docs/concepts/data-sources
+  - /docs/concepts/otel-concepts
+weight: 11
+default_lang_commit: a0ac78ec4ec9c360ea8f86481cf01bf2b82140f1
+---
+
+OpenTelemetry 的目的是收集、处理和导出**[信号][]**。
```
Comment (windsonsea) [inline content/zh/docs/concepts/signals/_index.md:11]:
```suggestion
OpenTelemetry 的目的是收集、处理和导出 **[信号][]** 。
```
The [preview](https://deploy-preview-5521--opentelemetry.netlify.app/zh/docs/concepts/signals/) looks not good.

**Review by windsonsea — COMMENTED** [review]
The [CI log](https://github.com/open-telemetry/opentelemetry.io/actions/runs/11668340300/job/32487780254?pr=5521) provides an error:

```
HASH	ERROR	content/zh/docs/concepts/signals/_index.md: git diff error (128)
or invalid hash a0ac78ec4ec9c360ea8f86481cf01bf2b82140f1. For details, use -v.
```

**Review by chalin — COMMENTED** [review]
@my-git9:

- Ensure that you rebase your PR and force push your updates.
- The hash code you use must be a valid hash code. You can use the current HEAD, [e7c30e9](https://github.com/open-telemetry/opentelemetry.io/commit/e7c30e90a49d961e8c4511e89187f656ad1cbfcf), as of the time of writing this.
- Also address the other GitHub action failures (file format and markdown linter)

**Comment by svrnm** [comment]
@open-telemetry/docs-zh-approvers PTAL!

File: content/zh/docs/concepts/signals/_index.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  - /docs/concepts/otel-concepts
+weight: 11
+default_lang_commit: e7c30e9
+---
+
+OpenTelemetry 的目的是收集、处理和导出**[信号][]**。
+信号是系统输出，描述了操作系统和平台上运行的应用程序的底层活动。
+信号可以是你希望在特定时间点测量的某项指标，如温度或内存使用率，
+也可以是贯穿分布式系统组件的事件，你希望对其进行跟踪。
+你可以将不同的信号组合在一起，从不同角度观察同一技术的内部运作。
```
Comment (reyang) [inline content/zh/docs/concepts/signals/_index.md:15]: I don't know how to translate "the same piece of technology", but I feel "同一技术" is very unclear and misleading.

Comment (windsonsea) [inline content/zh/docs/concepts/signals/_index.md:15]:
```suggestion
你可以将不同的信号组合在一起，从不同角度观察同一种技术的内部运作方式。
```
这样会不会通顺点

**Review by windsonsea — APPROVED** [review]

**Review by svrnm — APPROVED** [review]

**Pull request merged**